recipes/openscenegraph/all/conanfile.py

```diff
@@ -0,0 +1,538 @@
+from conans import CMake, ConanFile, tools
+import glob, os
+
+
```

Comment:
Since you are using components

```suggestion

required_conan_version = ">=1.28.0"

```